xml/System.Net.NetworkInformation/NetworkInterface.xml

```diff
@@ -561,28 +561,16 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The interface types are described in detail in the <xref:System.Net.NetworkInformation.NetworkInterfaceType> enumeration documentation.  
-  
- This property only returns a subset of the possible values defined in the <xref:System.Net.NetworkInformation.NetworkInterfaceType> enumeration. The possible values include the following:  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.Ethernet>  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.Fddi>  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.Loopback>  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.Ppp>  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.Slip>  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.TokenRing>  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.Unknown>  
-  
-   
-  
+ The interface types are described in detail in the <xref:System.Net.NetworkInformation.NetworkInterfaceType?displayProperty=nameWithType> enumeration documentation.  
+
+ In principle, this property can return any member of the <xref:System.Net.NetworkInformation.NetworkInterfaceType> enumeration. The specific value that it does return is dependent on the runtime conditions of the network topology. This means that:
+ 
+ - It is not possible to determine in advance which value the property returns for a particular network.
+ 
+ - It is possible that in practice on a particular network, the full range of enumeration values is not available. For example, on a gigabit Ethernet network, it is possible that the property returns <xref:System.Net.NetworkInformation.NetworkInterfaceType.Ethernet?displayProperty=nameWithType> rather than <xref:System.Net.NetworkInformation.NetworkInterfaceType.GigabitEthernet?displayProperty=nameWithType>.
+
 ## Examples  
- The following code example displays type information for all interfaces on the local computer.  
+ The following example displays type information for all interfaces on the local computer.  
```

Comment:
maybe replace "computer" with "machine"?

Reply:
>maybe replace "computer" with "machine"?

I'm not sure about change.  But we probably have a standard across all of our docs for proper word choice here.  Maybe it is "machine" instead of "computer". But we should pick whichever one we are using in other docs.

Reply:
From Microsoft style guide: 
"Most of the time, don't use machine to mean a computer. Use computer instead. It's OK to use machine in content for a technical audience and in content about virtualization to describe both physical machines and virtual machines."